app/worker/utils.go

```diff
@@ -15,17 +15,57 @@
 package worker
 
 import (
-	"fmt"
+	"os"
+	"regexp"
 	"strings"
 
 	"github.com/mendersoftware/workflows/model"
+	"github.com/thedevsaddam/gojsonq"
 )
 
+const (
+	workflowEnvVariable   = "env."
+	workflowInputVariable = "workflow.input."
+)
+
+var reExpression = regexp.MustCompile(`\$\{([^\}]+)\}`)
+var reExpressionOutput = regexp.MustCompile(`(.*)\.output\.(.*)`)
```

Comment:
real nitpicking mode on: if you have declared `workflowInputVariable` so why not declare `"output"` as well?

Reply:
Done